backend/dataall/modules/mlstudio/services/permissions.py

```diff
@@ -0,0 +1,44 @@
+"""
+Add module's permissions to the global permissions.
+Contains permissions for sagemaker ML Studio
+"""
+
+from dataall.db.permissions import (
+    ENVIRONMENT_ALL,
+    ENVIRONMENT_INVITED,
+    RESOURCES_ALL_WITH_DESC,
+    RESOURCES_ALL,
+    ENVIRONMENT_INVITATION_REQUEST,
+    TENANT_ALL,
+    TENANT_ALL_WITH_DESC
+)
+
+
+CREATE_SGMSTUDIO_USER = 'CREATE_SGMSTUDIO_USER'
+LIST_ENVIRONMENT_SGMSTUDIO_USERS = 'LIST_ENVIRONMENT_SGMSTUDIO_USERS'
+
+MANAGE_SGMSTUDIO_USERS = 'MANAGE_SGMSTUDIO_USERS'
+
+GET_SGMSTUDIO_USER = 'GET_SGMSTUDIO_USER'
+UPDATE_SGMSTUDIO_USER = 'UPDATE_SGMSTUDIO_USER'
+DELETE_SGMSTUDIO_USER = 'DELETE_SGMSTUDIO_USER'
+SGMSTUDIO_USER_URL = 'SGMSTUDIO_USER_URL'
+
+SGMSTUDIO_USER_ALL = [
+    GET_SGMSTUDIO_USER,
+    UPDATE_SGMSTUDIO_USER,
+    DELETE_SGMSTUDIO_USER,
+    SGMSTUDIO_USER_URL,
+]
+
+ENVIRONMENT_ALL.append(CREATE_SGMSTUDIO_USER)
+ENVIRONMENT_ALL.append(LIST_ENVIRONMENT_SGMSTUDIO_USERS)
+ENVIRONMENT_INVITED.append(CREATE_SGMSTUDIO_USER)
+ENVIRONMENT_INVITED.append(LIST_ENVIRONMENT_SGMSTUDIO_USERS)
+ENVIRONMENT_INVITATION_REQUEST.append(CREATE_SGMSTUDIO_USER)
```

Comment:
@nikpodsh I am not sure about this, maybe I understood it wrong, let's see :)
`ENVIRONMENT_ALL` = all permissions in the environment, given to environment Admin group
`ENVIRONMENT_INVITED` = all permissions that can actually be granted to invited groups in an environment. Used in `db.api.Environment#validate_permissions`, here we add the selectable + **some additional ones** and then check against `ENVIRONMENT_INVITED`
`ENVIRONMENT_INVITATION_REQUEST` = all permissions that are selectable to invited groups in an environment. Used in `backend.dataall.db.api.environment.Environment.list_group_invitation_permissions` to plot selectable permissions.

I understand that those **additional ones** are the ones that you want to get rid of, but we cannot do this without checking because we plot the results of `list_group_invitation_permissions` in the frontend

Reply:
Since we deleted this line in `environment.py`:
```
if permissions.CREATE_SGMSTUDIO_NOTEBOOK in g_permissions:
            g_permissions.append(permissions.LIST_ENVIRONMENT_SGMSTUDIO_NOTEBOOKS)
```

We will always have issue in this line:
```
if g_permissions not in permissions.ENVIRONMENT_INVITED:
       exceptions.PermissionUnauthorized(
                action='INVITE_TEAM', group_name=group, resource_uri=uri
            )
```
P.S. just noticed that we don't raise the error (just create it). And in many places too